templates/store/publisher-details.html

```diff
@@ -33,14 +33,15 @@ <h1 class="p-heading--two p-snap-heading__name">{{ publisher }}</h1>
             </span>
             {% endif %}
           </p>
+          <ul class="p-inline-list--middot u-float-left u-hide--large"></ul>
```

Comment:
Sorry I think we should apply some class to the "p-snap-heading" in a media query for smaller screens rather then include an empty element.

Reply:
I just made an update that removes the unnecesary margin and markup for this list. Would be great if you can double check it makes sense on snap detail pages as well.